lib/geolocation.js

```diff
@@ -0,0 +1,99 @@
+import _ from 'lodash';
+import log from './logger';
+import { exec } from 'teen_process';
+import { fs } from 'appium-support';
+
+const LYFT_SET_LOCATION = 'set-simulator-location';
+
+/**
+ * Set custom geolocation parameters for the given Simulator using LYFT_SET_LOCATION.
+ *
+ * @param {string} udid - The udid to set the given geolocation
+ * @param {string|number} latitude - The latitude value, which is going to be entered
+ *   into the corresponding edit field, for example '39,0006'.
+ * @param {string|number} longitude - The longitude value, which is going to be entered
+ *   into the corresponding edit field, for example '19,0068'.
+ * @return {boolean} True if the given geolocation was able to set.
+ */
+async function execLyftSetLocation (udid, latitude, longitude) {
```

Comment:
I would prefer this function to throw an exception in case of failure rather then returning a boolean result

Reply:
same below